### DISCUSSION TITLE
How the whole trakt import thing supposed to work?

### DISCUSSION THREAD

**Discussion by TheGuyWhoo** [umbrellaplug/umbrellaplug.github.io#204] [General]
Sometimes I get it to import my shows, other times it tells me addon folder is empty. What is it talking about? The trakt list isn't empty at all. Or is it referring to the library source folder? Isn't the whole point to import the shows into the folder? I am very confused. I did do the add folders thingy and I did restart kodi.

**Comment by umbrellaplug** [comment]
Anytime you do an import it looks at TV Shows and Movies folder for existing shows and movies before adding them. I then run a check for new strm files made during the list import and send kodi the command to update the library with those new shows. You will see that message sometimes for the folder being empty because nothing was added. It should be adding strm files then using your default scraper in Kodi to get meta for the added item and adding it to your Kodi library.
